cmd/autoscaler/main.go

```diff
@@ -100,7 +101,7 @@ func main() {
 	scaler := kpa.NewScaler(opt.ServingClientSet, opt.ScaleClientSet, logger, opt.ConfigMapWatcher)
 
 	controllers := []*controller.Impl{
-		kpa.NewController(&opt, paInformer, serviceInformer, endpointsInformer, multiScaler, collector, scaler, dynConfig),
+		kpa.NewController(&opt, paInformer, sksInformer, serviceInformer, endpointsInformer, multiScaler, collector, scaler, dynConfig),
 		hpa.NewController(&opt, paInformer, hpaInformer),
```

Comment:
🤔 Is the HPA controller going to have to do this too at some point?

Reply:
In the next PR, I think ...